### ISSUE TITLE
jrobin library not downloadable from unreachable glassfish maven repository

### ISSUE THREAD

**Issue by GoogleCodeExporter** [javamelody/javamelody#143]
```
What steps will reproduce the problem?
1. clean the local maven repository
2. create a maven project using javamelody-core as dependency
3. build it

What is the expected output? What do you see instead?

Should download javamelody-core and jrobin dependency, instead I get:

[ERROR] Failed to execute goal on project myproject: Could not resolve 
dependencies for project com.example:myproject:jar:0.1-SNAPSHOT: Failed to 
collect dependencies for [net.bull.javamelody:javamelody-core:jar:1.31.0 
(compile)]: Failed to read artifact descriptor for org.jrobin:jrobin:jar:1.5.9: 
Could not transfer artifact org.jrobin:jrobin:pom:1.5.9 from/to m.g.o-public 
(http://maven.glassfish.org/content/groups/public/): Error transferring file: 
Server returned HTTP response code: 503 for URL: 
http://maven.glassfish.org/content/groups/public/org/jrobin/jrobin/1.5.9/jrobin-
1.5.9.pom -> [Help 1]

What version of the product are you using? On what application server, JDK,
operating system?

javamelody 1.31.0, JDK 1.6, Mac OS X 10.6, Maven 3.0.3


Please provide any additional information below.

The glassfish maven repository server is not reachable, ideally there should 
not be any additional repositories in the javamelody POM in central, otherwise 
there should be the OpenNMS one instead, as in the jrobin home page: 
http://www.jrobin.org/index.php/Main_Page

    <repository>
      <id>opennms-repo</id>
      <name>OpenNMS Repository</name>
      <url>http://repo.opennms.org/maven2</url>
      <snapshots>
        <enabled>false</enabled>
      </snapshots>
    </repository>



```

Original issue reported on code.google.com by `marcello.teodori` on 29 Sep 2011 at 2:17
- Blocked on: #194
- Merged into: #194
